### ISSUE TITLE
could you provide a pretrained model of Inception-Resnet-v1 trained in imagenet dataset?

### ISSUE THREAD

**Issue by clscy** [davidsandberg/facenet#777]
Hi,
I want to train classifier using softmax loss with the Inception-Resnet-v1 model, could you provide a pretrained model of Inception-Resnet-v1 trained in imagenet dataset?
thanks.
